gigareview/console/models/ManuscriptsWorker.php

```diff
@@ -3,6 +3,7 @@
 namespace console\models;
 
 use \Yii;
+use common\models\Manuscript;
 
 class ManuscriptsWorker extends \yii\base\Component implements \yii\queue\JobInterface
```

Comment:
I'm not sure if we need this `ManuscriptsWorker` class that implements `\yii\queue\JobInterface` in addition to the `EMReportJob` class which already implements `\yii\queue\JobInterface`. You could think about moving the `parseManuscriptReport()` and `saveManuscripts()` functions into the `EMReportJob` class, and not have this class.

Reply:
Since `ManuscriptsWorker` class contains functions specific to the manuscript work, and it uses a generic `parseReport()` function in `EMReport` class that has already implemented `\yii\queue\JobInterface`, so ` \yii\queue\JobInterface` is removed from `ManuscriptsWorker` class.